tests/unit/rest.py

```diff
@@ -231,51 +231,58 @@ def test_none_return(self):
 
 @pytest.fixture()
 def validation_setup():
-    # There's a mix of PUT and POST in these; mixing it up may give us
-    # better coverage. The particulars of which calls get which methods are
-    # arbitrary.
+    # There's a mix of GET, PUT and POST in these; mixing it up may give us
+    # better coverage. The particulars of which PUT and POST calls get which
+    # methods are arbitrary, though GET calls have some specificity.
 
     # We have four kinds of calls we want to validate here:
 
     # 1. No arguments in the URL or body (no_args)
-    @rest.rest_call('POST', '/no/args', Schema({}))
+    @rest.rest_call(['GET','POST'], '/no/args', Schema({}))
```

Comment:
IIRC, PEP8 expects you to have a space after that comma. Want to run a linter on this before we merge it? Soon travis will do that for us, but...